core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -293,7 +293,7 @@ protected[controller] class RestAPIVersion_v1(
     val internalInvokerHealth = {
         (path("invokers") & get) {
             complete {
-                loadBalancer.getInvokerHealth()
+                loadBalancer.invokerHealth.map(_.mapValues(_.asString).toJson.asJsObject)
```

Comment:
i think this route is dead in open, and used in hotswap.
change the format if needed for convenience.